packages/wonder-blocks-button/src/components/button.md

```diff
@@ -452,7 +487,7 @@ const styles = StyleSheet.create({
                 setTimeout(resolve, 1000);
             })}
         >
-            Async action, client-side nav
+            beforeNav, client-side nav
```

Comment:
Probably it would be a good idea too to move the current async stories next to this description. Those stories are defined here: 

https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-button/src/components/button.stories.js#L356-L411